engine/class_modules/sc_death_knight.cpp

```diff
@@ -3604,6 +3590,14 @@ struct blood_plague_t : public death_knight_spell_t
     may_miss = may_crit = hasted_ticks = false;
 
     base_tick_time *= 1.0 + p -> talent.rapid_decomposition -> effectN( 1 ).percent();
+
+    if ( p -> legendary.superstrain -> ok() && (p -> specialization() == DEATH_KNIGHT_FROST || p -> specialization() == DEATH_KNIGHT_UNHOLY) )
```

Comment:
Can probably check if the spec isn't Blood, rather than checking if it's either of the other two.

Reply:
```suggestion
    if ( p -> legendary.superstrain -> ok() && ! p -> specialization() == DEATH_KNIGHT_BLOOD )
```